frontend/__tests__/components/utils/name-value-editor.spec.tsx

```diff
@@ -1,12 +1,10 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import { DragDropContext } from 'react-dnd';
-import HTML5Backend from 'react-dnd-html5-backend';
-
 import { NameValueEditor } from '../../../public/components/utils/name-value-editor';
+import withDragDropContext from '../../../public/components/utils/drag-drop-context';
 
-describe(NameValueEditor.displayName, () => {
-  const Editor = DragDropContext(HTML5Backend)(NameValueEditor);
+describe('Name Value Editor', () => {
+  const Editor: React.FC<NameValueEditorProps> = withDragDropContext(NameValueEditor as any);
```

Comment:
`NameValueEditor` is already wrapped in `withDragDropContext` so theres no need to do it again

Reply:
Wow, I totally missed that !